cmd/start.go

```diff
@@ -70,11 +77,23 @@ $ %s start demo-path2 --max-tx-size 10`, appName, appName)),
 				return err
 			}
 
+			timeInterval := viper.GetDuration(flagTimeInterval)
+
+			go func() {
+				for {
+					// run for every 60 seconds
+					if err := UpdateClientsFromChains(c[src], c[dst]); err != nil {
+						fmt.Printf("Error in updating clients....%v", err)
```

Comment:
I think this should probably return an error, we don't want this silently failing. Maybe we can add some retries before returning an error